docs/src/components/Playground/renderConfig.ts

```diff
@@ -24,6 +24,7 @@ export const imports = {
   react: React,
   'react-dom': ReactDOM,
   'react-fela': ReactFela,
+  'react-scripts': 'latest',
```

Comment:
I suggest to place this thing in `createPackageJson.ts` instead:

```jsx
const dependencies = {
  ..._.mapValues(imports, () => 'latest'),
  // required to enable all features due old templates in https://github.com/codesandbox/codesandbox-importers
  // https://github.com/stardust-ui/react/issues/1519
  'react-scripts': 'latest',
}
```

If we will place this here it can be resolved to string in our examples 💥